src/V3Graph.h

```diff
@@ -172,7 +172,106 @@ class V3Graph VL_NOT_FINAL {
 
 //============================================================================
 
+// Holds list of types. For use mainly in compile-time code generation.
+template <typename... TN>
+struct VTypeList {
+    template <typename... UN>
+    constexpr VTypeList<TN..., UN...> operator+(VTypeList<UN...>) const {
+        return {};
+    }
+};
+
+namespace internal_vl {
```

Comment:
```suggestion
namespace VInternal {
```

Reply:
Not sure I like "internal" here; is it really more internal than anything else in Verilator? Maybe something with "type info" or even "graph type info"?

Reply:
I put it in "internal" namespace only because it does not really fit in V3Graph.h as a "public" code. What about moving this to a header dedicated for utility code?

Reply:
Personally I'd be okay with some sort of RTTI-dedicated header.

Reply:
I ended up moving the code to V3Rtti.h, but still put 3 functions in `V3RttiInternal` namespace. I've opted for `V3Rtti` prefix for consistency with the file name. The functions inside this namespace are strictly an implementation detail of "public" code and shouldn't be used anywhere else.